pages/key/create.js

```diff
@@ -149,20 +158,31 @@ export default function CreateKey() {
       {state.step === steps.PASSWORD && (
         <AuthLayout>
           <AuthLayout.Normal>
-            <Flex width="100%">
+            <Flex
+              direction={['column', 'initial']}
+              align={['center', 'initial']}
+              width="100%"
+            >
               <Avatar address={state.address} />
               <Flex
                 direction="column"
+                align={['center', 'initial']}
                 justify="center"
                 flex="1"
                 style={{marginLeft: rem(20)}}
               >
-                <SubHeading color="white">
-                  {t('Create password to encrypt your account')}
-                </SubHeading>
+                <Box display={['none', 'inherit']}>
+                  <SubHeading color="white">
+                    {t('Create password to encrypt your account')}
+                  </SubHeading>
+                </Box>
 
-                <Flex justify="space-between">
-                  <Text color="xwhite.050" fontSize={rem(14)}>
+                <Flex justify="space-between" w={['85%', '100%']}>
+                  <Text
+                    wordBreak={['break-all', 'initial']}
+                    color="xwhite.050"
+                    fontSize={rem(14)}
```

Comment:
`fontSize={3.5}`, `fontSize={"14px"}`, `fontSize="mdx"`, please try to avoid `rem`